doc/classes/VoxelTool.xml

```diff
@@ -100,12 +108,14 @@
 			<return type="bool" />
 			<param index="0" name="box" type="AABB" />
 			<description>
+				Returns false if the blocks within `box` are not fully loaded. Not implemented in this class, because it only makes sense if editing with [VoxelToolTerrain] or [VoxelToolLodTerrain]
```

Comment:
Haha, yeah, I wrote this looking at base implementation without realizing it won't appear in child class docs. I'm new to this xD
If links to main docs will work, maybe it can mention "Useful to check if edited area is within bounds and in LOD0 in case of LOD terrain" and link back to the section about this in Scripting.